src/index.ts

```diff
@@ -66,37 +90,129 @@ export class MtLinkSdk {
+   *
+   * @remark
+   * ⚠️ You must initialize the SDK via the {@link init} before calling this API.
+   *
+   * ⚠️ For legal reasons, you have to set up your app's terms and conditions URL to use the onboard API.
+   * Please ask your Moneytree representative for more details.
+   *
+   * @throws If the `email` property is not set (via {@link init} or in the `options` parameter of this function).
+   */
   public onboard(options?: OnboardOptions): void {
     onboard(this.storedOptions, options);
   }
 
+  /**
+   * This method generates a URL for guest onboarding.
+   *
+   * This API has exactly the same parameters as {@link onboard}, the only difference being that it returns an URL
+   * instead of opening immediately with `window.open`.
+   */
   public onboardUrl(options?: OnboardUrlOptions): string {
     return onboardUrl(this.storedOptions, options);
   }
 
+  /**
+   * Logout current user from Moneytree.
+   */
   public logout(options?: LogoutOptions): void {
     logout(this.storedOptions, options);
   }
 
+  /**
+   * This method generates a URL to log out the guest.
+   *
+   * This API has exactly the same parameters as {@link logout}, the only difference being that it returns an URL
+   * instead of opening immediately with `window.open`.
+   */
   public logoutUrl(options?: LogoutUrlOptions): string {
     return logoutUrl(this.storedOptions, options);
   }
 
+  /**
+   * This is a method to open various services provided by Moneytree such as Moneytree account settings, Vault etc.
+   *
+   * Pass `serviceId: 'link-kit'` to open the LINK Kit service.
+   *
+   * @remark ⚠️ calling this API before calling {@link init} will open the services view without branding (company logo etc.)
+   */
   public openService(serviceId: 'link-kit', options?: LinkKitOpenServiceOptions): void;
+  /**
+   * Pass `serviceId: 'myaccount'` to open the MyAccount service.
+   *
+   * Open different MyAccount sub-pages by passing the {@link MyAccountServiceTypes} for all possible options.
+   *
+   * @remark ⚠️ calling this API before calling {@link init} will open the services view without branding (company logo etc.)
+   */
   public openService(serviceId: 'myaccount', options?: MyAccountOpenServiceOptions): void;
+  /**
+   * Pass `serviceId: 'vault'` to open the Vault service.
+   *
+   * Depending on the Vault sub-page you want to open, you can pass the following options:
+   * - `serviceList`: opens the vault service list page, pass {@link VaultOpenServiceViewServiceList} as options.
+   * - `serviceConnection`: opens the vault service connection page, pass {@link VaultOpenServiceViewServiceConnection} as options.
+   * - `connectionSetting`: opens the vault connection setting page, pass {@link VaultOpenServiceViewConnectionSetting} as options.
+   * - `customerSupport`: opens the vault customer support page, pass {@link VaultOpenServiceViewCustomerSupport} as options.
```

Comment:
Given that "Vault" is branding, it seems appropriate to always write it capitalized like on lines 206 and 208. So in the description for all these options I would suggest replacing "vault" with "Vault".